web/src/js/components/common.js

```diff
@@ -131,4 +131,26 @@ export var Splitter = React.createClass({
             </div>
         );
     }
-});
\ No newline at end of file
+});
+
+export class ToggleComponent extends React.Component{
+  render(){
+    let toggleIconClass = this.props.checked ? "fa-check-square-o" : "fa-square-o";
+    let toggleButtonClass = this.props.checked ? "btn-primary" : "btn-default";
+    return (
+    <div
+      className={"btn-option btn " + toggleButtonClass}
+      onClick={this.props.onToggleChanged}>
+      <i className={"fa " + toggleIconClass}> 
```

Comment:
i did this because then the text is in one line with the icon, otherwise there is a little ugly space gap